src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -461,6 +462,10 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
   }
 
   _syncCurrentTime(): void {
+    if (SEEK_AFTER_FOCUS) {
+      this._videoElement.currentTime -= 0.1;
```

Comment:
can't we do this in the `_handleDecodeError` when we set back current time?

Reply:
@OrenMe 
I tried, as it is more logical to put it there, however, the weird part of this issue, that the player never detect this issue as a decode error, or any error in fact, the player is just getting stuck with no capability of pausing or playing.

![image](https://user-images.githubusercontent.com/11364117/80791265-cfc0d600-8b99-11ea-9141-8d80bdf34cea.png)

As you can see in the above pic, for some reason we never enter `_handleDecodeError` if we did we would have [log](https://github.com/kaltura/playkit-js/blob/84b54a1023516f3595970990dc72cdb2eabce7cb/src/engines/html5/media-source/adapters/native-adapter.js#L332) it.